tests/behavioural/features/v2/draft_get.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Get draft by id V2
+
+  Background: Reset database
+    Given prepare for tests using 'mock' services
+
+  Scenario Outline: A Respondent saves a draft and reads it
+      And sending from respondent to internal <user>
```

Comment:
In the V2 feature files I think a lot of these should be split out, I don't think having 5+ 'and' is helpful. If we're using that many then they are really different scenarios

Reply:
Agree with complexity is potentially an issue , but not sure how to make them easier given the complexity around threads

Reply:
I agree with @dcdarrell9 on this one - is there a way to specify tests against v1/v2 without the need to have so many `and` clauses?

Reply:
The two things are separate . Some of the tests can have reduced numbers of steps by combining some . But in the case of a conversation there are many things to do. Set up the user , set up the two and from , send a message , switch users , read a message , respond , possibly save a draft . I think some can be cleaned up on a case by case basis but there is no one size fits all .